cmd/junod/bech32.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+// DONTCOVER
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/types/bech32"
+)
+
+var flagBech32Prefix = "prefix"
+
+// get cmd to convert any bech32 address to an juno prefix.
+func ConvertBech32Cmd() *cobra.Command {
```

Comment:
Was looking for this the other day. lol

Reply:
I should have probably moved it to It’s own PR oops